cron.sh

```diff
@@ -11,12 +11,12 @@ sbt "runMain apps.ConvertUpdates $(tail -n1 GND-lastSuccessfulUpdate.txt)"
 
 # copy and index to stage, and check:
```

Comment:
I understand that this script would start updates. If I would want to start a fulldump this script would have to be manipulated like "cp $fulldump ..." and "sbt "runMain apps.Index baseline", same goes with the entityfacts resulting of #216, yes ? If so, however, it would be nice to have those procedures already in the script, and the script would take parameters and switch "updates", "baseline" and "entityfacts". Possibly anticipating your objection "only updates are regular and thus to be croned":
1. it would nonetheless be easier to start those other index creations, and 2. even if they are not regular provided one could schedule them if it's known when they will be provided at least and 3. they will become regularly provided in future I assume, 4. the automatic  test at the end of the script may also be applied
First point is the most important so.
What do you think?

Reply:
I agree that it would be nice to have automated (or automatable/one-step executable) workflows for EntityFacts and GND baseline conversion and indexing.

Since this will involve quite some work (handling the three cases in the script, getting the baseline and entityfacts data, making it possible to pass details as parameters instead of using the conf file, see README), I suggest we open a separate issue for this.

Reply:
Ok, opened #218.